ras_backstage/error_handlers.py

```diff
@@ -25,6 +26,19 @@ def api_error_method(error):
     return jsonify(error_json), status_code
 
 
+@app.errorhandler(JSONDecodeError)
+@api.errorhandler(JSONDecodeError)
```

Comment:
Does `app.errorhandler` not mean the api blueprint is covered too? If http://flask.pocoo.org/docs/0.12/api/#flask.Blueprint.app_errorhandler looks like it covers it

Reply:
This is a bug in flask-restplus. Currently the error handling wont work unless you have both of these here